upstream/reconcile.go

```diff
@@ -8,27 +8,36 @@ import (
 	"net/http"
 	"net/url"
 
+	"github.com/flanksource/duty/models"
 	"github.com/flanksource/incident-commander/api"
 	"github.com/flanksource/incident-commander/db"
 )
 
 // SyncWithUpstream sends all the missing resources to the upstream.
 func SyncWithUpstream(ctx context.Context) error {
+	jobHistory := models.NewJobHistory("SyncWithUpstream", "", "")
+	_ = db.PersistJobHistory(jobHistory.Start())
+	defer db.PersistJobHistory(jobHistory.End())
```

Comment:
This needs to be wrapped in an anonymous function.

Reply:
Ahh, good catch